packages/@aws-cdk/aws-ecs-patterns/test/ec2/l3s-v2.test.ts

```diff
@@ -270,26 +269,6 @@ describe('When Application Load Balancer', () => {
     });
   });
 
-  test('set vpc instead of cluster', () => {
-    // GIVEN
-    const stack = new Stack();
-    const vpc = new Vpc(stack, 'VPC');
-
-    // WHEN
-    new ApplicationMultipleTargetGroupsEc2Service(stack, 'Service', {
-      vpc,
-      memoryLimitMiB: 1024,
-      taskImageOptions: {
-        image: ContainerImage.fromRegistry('test'),
-      },
-    });
-
-    // THEN - stack does not contain a LaunchConfiguration
-    const template = SynthUtils.synthesize(stack, { skipValidation: true });
```

Comment:
This is one of the three removed tests that were all removed for the same reason. With Assertions, there is no way to skip synthesis validation (as far as I know!), so we can't do this test. And, even if we could, it seems completely pointless because a customer could never synthesize a template like this because of the validation.